extensions/amp-mowplayer/0.1/amp-mowplayer.js

```diff
@@ -282,24 +289,16 @@ class AmpMowplayer extends AMP.BaseElement {
   }
 
   /** @override */
-  play(unusedIsAutoplay) {
-    this.sendCommand_('playVideo');
-  }
+  play() {}
```

Comment:
If you remove these methods, nothing else controlling playback will work anymore (including docking).